.prettierignore

```diff
@@ -0,0 +1 @@
+**
```

Comment:
Why?

Reply:
Because they don't use prettier! And it's messing up the change list! Alternatively, we run it on the whole codebase at once. This may make it harder to sync upstream though!